coursework2/src/main/java/oursework2/controller/ExamController.java

```diff
@@ -0,0 +1,23 @@
+package oursework2.controller;
+
+import org.springframework.web.bind.annotation.*;
+import oursework2.model.Question;
+import oursework2.service.ExaminerService;
+import oursework2.service.ExaminerServiceImpl;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam")
+public class ExamController {
+    private final ExaminerServiceImpl examinerService;
```

Comment:
```suggestion
    private final ExaminerService examinerService;
```
по интерфейсу надо делать инъекцию, а не реализации (см. полиморфизм, L в SOLID)